pom.xml

```diff
@@ -42,19 +42,14 @@
     </distributionManagement>
     <properties>
         <build-helper-maven-plugin.version>3.3.0</build-helper-maven-plugin.version>
-        <commons-beanutils.version>1.9.4</commons-beanutils.version>
         <commons-codec.version>1.15</commons-codec.version>
-        <findbugs-jsr305.version>3.0.2</findbugs-jsr305.version>
-        <http-components.version>4.5.13</http-components.version>
-        <http-components-core.version>4.4.15</http-components-core.version>
         <java-poet.version>1.13.0</java-poet.version>
         <javax-validation-api.version>2.0.0.Final</javax-validation-api.version>
         <junit.version>5.9.1</junit.version>
         <lombok.version>1.18.24</lombok.version>
         <lombok-maven-plugin.version>1.18.20.0</lombok-maven-plugin.version>
         <mockserver.version>5.14.0</mockserver.version>
         <reflections.version>0.10.2</reflections.version>
-        <site-skin.version>1.5</site-skin.version>
```

Comment:
This is a typo, introduced in 44c5bd80398e30064d6f5f3620ee8e64803f38b6, for `site-skin-version`, used in src/site/site.xml. No one noticed because we don’t publish a site for this module. I think we should delete src/site/site.xml (because it is unused), fix this property typo (because it could theoretically be used), or keep this property as it was (because removing unused properties is unrelated to this ticket). Deleting only the property exacerbates the bitrot.

Reply:
Let’s do the first option

Reply:
Let’s do the first option, I’ll delete `src/site/site.xml`